Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -82,6 +82,9 @@ import BraintreeCore
     /// Optional: If set to `true`, this enables the Checkout with Vault flow, where the customer will be prompted to consent to a billing agreement during checkout. Defaults to `false`.
     public var requestBillingAgreement: Bool
 
+    /// Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
+    public var userAuthenticationEmail: String?
```

Comment:
I think we decided this should _only_ live in `BTPayPalNativeCheckoutRequest` as adding it here would not work for the web based flows. I think we can remove this.

Reply:
+1